src/locales/en.json

```diff
@@ -257,10 +263,12 @@
     "normal_voice_short": "Normal",
     "oneTime": "A gathering, activity, or errand",
     "outdoor": "Outdoor",
+    "outdoor_breakdown": "for outdoor",
     "p100_mask_short": "P100 respirator",
     "p100_mask_theirs": "P100 respirator, NIOSH-certified reusable, unclear if outflow valve is covered",
     "p100_mask_yours": "P100 respirator, NIOSH-certified reusable, with covered outflow valve",
     "partially_enclosed": "A space with one or more sides open to the outdoors",
+    "partially_enclosed_breakdown": "for partially enclosed space",
```

Comment:
for increased airflow from a space with one or more sides open to the outdoors.

Reply:
long